src/Computer/GlideComputerAirData.cpp

```diff
@@ -190,13 +189,15 @@ GlideComputerAirData::CurrentThermal(const MoreData &basic,
                                      OneClimbInfo &current_thermal)
 {
   if (circling.climb_start_time.IsDefined()) {
-    current_thermal.start_time = circling.climb_start_time;
+    if (current_thermal.start_time != circling.climb_start_time) {
```

Comment:
Do not use `==` or `!=` to compare floating point values. (I don't understand what this change is trying to do.)

Reply:
It just triggers the very first call. Ther "enter climb" condition.